omnibus/package-scripts/chef-workstation/postinst

```diff
@@ -25,7 +25,21 @@ else
     PREFIX="/usr"
 fi
 
-binaries="chef-run berks chef chef-cli chef-analyze chef-apply chef-shell chef-solo chef-vault cookstyle delivery foodcritic inspec kitchen knife ohai push-apply pushy-client pushy-service-manager chef-client"
+binaries="chef-run berks chef chef-cli chef-analyze chef-apply chef-shell chef-solo chef-vault cookstyle delivery foodcritic inspec kitchen knife ohai push-apply pushy-client pushy-service-manager chef-client hab"
+
+old_hab_path=$(which hab)
+
+if [[ "$old_hab_path" != "" ]]; then
+    loc=$(readlink "$old_hab_path") # If it's not a link readlink gives back the same path.
+    if [[ $loc != *"chef-workstation"* ]] ; then
+      echo ""
```

Comment:
Phrasing check request for @chef/docs-team. 

Context:  This is expected to be a minimum occurrence/edge case.   This message will be shown after installing the updated package on mac/linux, if the operator has already installed Habitat  directly on the node.  When that happens, we prefer for customers to receive future Hab updates through the Workstation updates, in the same way we deliver Infra, Inspec, etc as part of Workstation. 



If a node has both Workstation and Habitat installed,  it simplifies life for the operator and for us if future updates to habitat are provided via Workstation updates.  Nothing will break if they go back to updating habitat directly from stable, current, etc - but each Workstation install will overwrite it.  
If habitat